src/files/context-menu/ContextMenu.js

```diff
@@ -61,7 +61,7 @@ class ContextMenu extends React.Component {
         <DropdownMenu
           top={-8}
           arrowMarginRight='11px'
-          left='calc(100% - 200px + 0.5rem)'
```

Comment:
Without this, the placement of the ContextMenu in preview mode is wrong.

Reply:
It works here :O

Reply:
One minor nitpick, with this PR when you right-click a file to open the context menu, if you then left click on a file/folder it will enter its preview mode or navigate to that folder, keeping the ContextMenu open.

In the live version this doesn't happen, if the ContextMenu is opened and you left-click anywhere, the ContextMenu gets closed, as I think that's the expected behaviour.

Reply:
![image](https://user-images.githubusercontent.com/5447088/57084457-42cb0e80-6cf2-11e9-8e44-0b235f66c323.png)

Notice that I changed the place where ContextMenu is, so it probably won't need it now. That's why I removed it.

Reply:
> It works here :O

As you can see it's not centered to the middle dot:

> <img width="255 " alt="ctx" src="https://user-images.githubusercontent.com/33324750/57084519-5eceb000-6cf2-11e9-8ed8-15ea4a18f941.png">

Reply:
Perhaps I should add like some pixels there to center it, but not that much.

Reply:
But if you do it will ruin the alignment when not in preview mode I think (in the files list).

Reply:
@fsdiogo could you take a look now, please? 😃

Reply:
Niiice, working good!